disnake/types/channel.py

```diff
@@ -140,8 +140,6 @@ class ThreadChannel(_BaseChannel, _ThreadChannelOptional):
     nsfw: bool
     last_message_id: Optional[Snowflake]
     rate_limit_per_user: int
-    message_count: Optional[int]
-    total_message_sent: Optional[int]
     member_count: int
```

Comment:
@shiftinv why were these lines deleted? do we have the thread typings defined twice?

Reply:
> @shiftinv why were these lines deleted? do we have the thread typings defined twice?

If they're already in `_ThreadChannelOptional` we don't need them in `ThreadChannel` too no?